### PR TITLE
Add json output

### PR DESCRIPTION
**Pull request by pguelpa**

### CODE REVIEW

File: spec/lib/mulder/formats/json_spec.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    it 'prints json encoded output to the stream' do
+      instances = [{ foo: :bar }]
+      mocked_instances = mock(collect: instances)
+
+      io = StringIO.new()
+      formatter = described_class.new(mocked_instances)
+      formatter.stream = io
+
+      formatter.output
+      expect(io.string).to eq(instances.to_json)
```
Comment (akatz) [inline spec/lib/mulder/formats/json_spec.rb]: it's always weird to test the implementation with the same method the implementation uses. I'd rather have a string expected here.

**Comment by akatz** [comment]
Minor comment but otherwise LGTM.

File: spec/lib/mulder/formats/json_spec.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    it 'prints json encoded output to the stream' do
+      instances = [{ foo: :bar }]
+      mocked_instances = mock(collect: instances)
+
+      io = StringIO.new()
+      formatter = described_class.new(mocked_instances)
+      formatter.stream = io
+
+      formatter.output
+      expect(io.string).to eq('[{"foo":"bar"}]')
```
Comment (pguelpa) [inline spec/lib/mulder/formats/json_spec.rb]: @akatz - Updated to use a string

**Comment by akatz** [comment]
LGTM :shipit:

**Comment by itsmeduncan** [comment]
I am usually against bumping the version in the changeset PR, but :boat:

**Pull request merged**